### DISCUSSION TITLE
Outdated Firefox logo

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#141] [Content]
Is there a reason for an outdated Firefox logo being used in some examples?

**Comment by teoli2003** [comment]
Yes, the example was not updated when the logo changed.

To prevent this kind of extra work, we should replace the logo with something that won't change in the future. So we should avoid logos (MDN, Firefox, …) in examples.

@Rumyra You told us yesterday that somebody with design abilities will start soon. Maybe they can design a logo-free replacement that we can use over the place? We always lose time finding/thinking about images for examples.

**Comment by Rumyra** [comment]
This is a great idea @teoli2003 

@MrBrain295 do you have a few links where these examples are?

**Reply by MrBrain295** [reply]
[This](https://github.com/mdn/dom-examples/blob/master/webgl-examples/tutorial/sample7/cubetexture.png) was the first one I saw but there are more in the webgl examples.

**Comment by Rumyra** [comment]
Closing this as we should be able to tackle it via the new asset repo and rolling that out across examples 👍
